api/util.py

```diff
@@ -0,0 +1,259 @@
+import asyncio
+from typing import Union, Tuple, List
+
+import cachetools
+import datetime
+from fcntl import fcntl, F_GETFL, F_SETFL
+import koji
+import logging
+import os
+import shlex
+import subprocess
+from threading import RLock
+import time
+from api.kerberos import do_kinit
+import functools
+
+logger = logging.getLogger()
+
+
+def please_notify_art_team_of_error(so, payload):
+    dt = datetime.datetime.today().strftime('%Y-%m-%d-%H-%M-%S')
+    so.snippet(payload=payload,
+               intro='Sorry, I encountered an error. Please contact @art-team with the following details.',
+               filename=f'error-details-{dt}.txt')
+
```

Comment:
this doesn't look like it belongs. true for much of this file... check if there are any actual references and cut out the deadwood

Reply:
Fixed and removed.